var/spack/repos/builtin/packages/upcxx/package.py

```diff
@@ -245,16 +245,12 @@ def test_install(self):
             make("run-tests", "NETWORKS=smp")  # runs tests for smp backend
         make("tests-clean")  # cleanup
 
-    def test(self):
-        # run post-install smoke test:
+    def test_upcxx_install(self):
+        """checking UPC++ compiler+link for all installed backends"""
```

Comment:
Minor quibble - UPC++ is a library and does not technically provide a compiler, this string was/is referring to compilation using the UPC++ header files (and subsequent linkage against the libraries).

```suggestion
        """checking UPC++ compile+link for all installed backends"""
```